api/alert_channel_datadog.go

```diff
@@ -0,0 +1,47 @@
+//
+// Author:: Vatasha White (<vatasha.white@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+// GetDatadog gets a single instance of a Datadog alert channel
+// with the corresponding integration guid
+func (svc *AlertChannelsService) GetDatadog(guid string) (response DatadogAlertChannelResponseV2, err error) {
+	err = svc.get(guid, &response)
+	return
+}
+
+// UpdateDatadog updates a single instance of a Datadog integration on the Lacework server
+func (svc *AlertChannelsService) UpdateDatadog(data AlertChannel) (response DatadogAlertChannelResponseV2, err error) {
+	err = svc.update(data.ID(), data, &response)
+	return
+}
+
+type DatadogDataV2 struct {
+	APIKey      string `json:"apiKey"`
+	DatadogSite string `json:"datadogSite,omitempty"`
+	DatadogType string `json:"datadogType,omitempty"`
```

Comment:
Is omitempty needed here? Schema says they are required fields

Reply:
There is a default value which is why I have omitempty